jvm/src/streamparse/commands/kill_topology.clj

```diff
@@ -30,7 +30,7 @@
   (let [[opts args banner]
          (cli args
               ["-h" "--help" "Show this help screen." :flag true :default false]
-              ["-w" "--wait" "Amount of time to wait for topology to gracefully stop." :default 10 :parse-fn #(Integer/parseInt %)]
+              ["-w" "--wait" "Amount of time to wait for topology to gracefully stop." :parse-fn #(Integer/parseInt %)]
```

Comment:
FWIW, unless it's changed in 0.9, this is *definitely* the default behavior of the `storm kill` CLI command. I've sat at the console watching the minutes tick by far too many times.